book/tutorials/microstructure/microstructure-tutorial.ipynb

```diff
@@ -0,0 +1,55 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Microstructure Tutorial\n",
+    "\n",
+    "Snow microstructure is important - it governs both visible and radar remote sensing to snow cover. \n",
+    "\n",
+    "Snow microstructure is controlled by other snow properties, such as snow temperature, snow height and snow liquid water content. \n",
+    "\n",
+    "# SnowEx Microstructure Measurement Background\n",
```

Comment:
JupyterBook parses markdown headers for website navigation. You can see how this is rendered on the preview link in this pull_request:  https://deploy-preview-34--snowex-hackweek.netlify.app/tutorials/microstructure/microstructure-tutorial.html . Long story short, you want just a single header (#Microstructure Tutorial, and then subsquent subheaders for example:
```
# Microstructure Tutorial
## Background
### Background subsection
## Data
```

Reply:
got it! just pushed a fix.